src/components/Superfest/SuperfestPresentedBy/SuperfestPresentedByBox.tsx

```diff
@@ -0,0 +1,35 @@
+'use client';
+
+import { usePathname } from 'next/navigation';
+import { SuperfestPresentedBy } from 'src/components/illustrations/SuperfestPresentedBy';
+import { JUMPER_FEST } from 'src/const/urls';
+import { SuperFestPoweredContainer } from './SuperfestPresentedByBox.style';
+
+export const SuperfestPresentedByBox = () => {
+  const currentPath = usePathname();
+  const isSuperfest = currentPath?.includes(JUMPER_FEST);
+
+  const handleClick = () => {
+    // openInNewTab(lifiUrl);
+  };
+
+  return (
+    <>
+      {isSuperfest ? (
+        <SuperFestPoweredContainer fixedPosition={false}>
+          <a
+            className={'link-superfest'}
+            onClick={handleClick}
+            href={'https://www.optimism.io/'}
```

Comment:
What do you think about such links living into url.ts?

Reply:
I will.